crates/wasmtime/src/runtime/vm/table.rs

```diff
@@ -526,17 +529,18 @@ impl Table {
     /// this unsafety.
     pub unsafe fn grow(
         &mut self,
-        delta: u32,
+        delta: u64,
         init_value: TableElement,
         store: &mut dyn Store,
-    ) -> Result<Option<u32>, Error> {
+    ) -> Result<Option<usize>, Error> {
         let old_size = self.size();
 
         // Don't try to resize the table if its size isn't changing, just return
         // success.
         if delta == 0 {
             return Ok(Some(old_size));
         }
+        let delta: usize = delta.try_into().unwrap();
```

Comment:
Here this'll want to return an error if this returns `Err` rather than panicking